estimagic/optimization/optimize.py

```diff
@@ -156,51 +135,49 @@ def minimize(
     Args:
         criterion (function or list of functions):
             Python function that takes a pandas DataFrame with parameters as the first
-            argument and returns a scalar floating point value.
-
+            argument. Supported outputs are:
+                - scalar floating point
+                - np.ndarray: contributions for the tao Pounders algorithm.
+                - tuple of a scalar floating point and a pd.DataFrame:
+                    In this case the first output is the criterion value.
+                    The second output are the comparison_plot_data.
+                    See :ref:`comparison_plot`.
+                    .. warning::
```

Comment:
Such warnings are not necessary. For large things I would make issues. If you want to tackle them, put them on your to-do list.

Reply:
Done in c56364a71dbbaf0122f4f136ffaedbeaa49b65cd.